contrib/database/sql/conn_test.go

```diff
@@ -0,0 +1,64 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package sql
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+)
+
+func TestWithSpanTags(t *testing.T) {
+	mt := mocktracer.Start()
+	defer mt.Stop()
+
+	// set tags
+	ctxTags := map[string]string{
+		"ctx_tag1": "value1",
+		"ctx_tag2": "value2",
+		"ctx_tag3": "value3",
+	}
+	ctx := WithSpanTags(context.Background(), ctxTags)
+	query := "SELECT 1"
+	startTime := time.Date(2019, 1, 1, 1, 0, 0, 0, time.UTC)
+
+	tp := &traceParams{
```

Comment:
Not like this. We want to use the real public API, not the private methods. You can get inspired from the rest of the tests. For example here: https://github.com/DataDog/dd-trace-go/blob/a59475992ec658c954de1e89fff55b4bfef54570/contrib/internal/sqltest/sqltest.go#L83-L96

Reply:
I didn't look this. thank you so much :bow: